libraries/c_sdk/standard/mqtt/src/private/iot_mqtt_internal.h

```diff
@@ -1148,7 +1148,7 @@ bool IotMqtt_RemoveSubscription( _mqttSubscription_t * pSubscriptionArray,
  *
  */
 void IotMqtt_RemoveAllMatches( _mqttSubscription_t * pSubscriptionArray,
-                               _packetMatchParams_t * pMatch );
+                               const _packetMatchParams_t * pMatch );
```

Comment:
why to remove const here?

Reply:
I have added the const qualifier as the matching params should not change.